stanford_fellowship.ds.inc

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * @file
+ * stanford_fellowship.ds.inc
+ */
+
+/**
+ * Implements hook_ds_layout_settings_info().
+ */
+function stanford_fellowship_ds_layout_settings_info() {
+  $export = array();
+
+  $ds_layout = new stdClass();
+  $ds_layout->api_version = 1;
+  $ds_layout->id = 'node|stanford_fellowship|full';
+  $ds_layout->entity_type = 'node';
+  $ds_layout->bundle = 'stanford_fellowship';
+  $ds_layout->view_mode = 'full';
```

Comment:
I have reverted this module several times and I don't this came through properly. The full content view mode is a flat list of fields. Do you have an example of what it should be so I can help figure out what went wrong?